indexing-service/src/main/java/org/apache/druid/indexing/common/task/HadoopIndexTask.java

```diff
@@ -202,7 +202,7 @@ public String getType()
   @Override
   public Set<ResourceAction> getInputSourceResources()
   {
-    return Collections.singleton(new ResourceAction(new Resource(ResourceType.EXTERNAL, "hadoop"), Action.READ));
```

Comment:
I think Hadoop's task type is called `index_hadoop` (and not `hadoop` - see https://druid.apache.org/docs/latest/ingestion/hadoop.html#task-syntax and the method `getType()` above).